.github/mergify.yml

```
@@ -1,9 +1,11 @@
 queue_rules:
-  - name: urgent
+  - name: hotfix
     allow_inplace_checks: False
     allow_checks_interruption: False
-    speculative_checks: 2
-    batch_size: 2
+    speculative_checks: 3
```

Question: do we actually want to run any checks in parallel? It might exceed quotas.

This is the maximum number of checks to run in parallel in the queue. Speculative checks are cancelled and re-run when a PR merges to main, so they don't actually speed up merging.
```suggestion
    speculative_checks: 1
```

https://docs.mergify.com/actions/queue/#queue-rules